### PR TITLE
ThrownPotionData - fix syntax not working

### PR DESCRIPTION
**Pull request by ShaneBeee**
### Description
This PR fixes ThrownPotionData not working. When attempting to spawn a thrown potion it would throw errors. This was first due to the lang file being incorrect. 
After fixing that, had to do a whole whack-a-doodle of fixes to make this work properly.

There are some bugs in 1.12.2 and below that throw errors when attempting to use a thrown potion. Silly Bukkit. The API states you can use a thrown potion, but I think someone did some errors in CB. 

At the same time I also fixed a lang issue with dropped item, this small fix allows you to spawn a dropped item of any time (kinda useless, can use the drop item effect, but figured I should fix it anyways, whilst I'm at it)

## NOTE:
The failure is due to precious stones, Im guessing their repo is down.

---
**Target Minecraft Versions:** any
**Requirements:** none
**Related Issues:** #2891

### CODE REVIEW

File: src/main/java/ch/njol/skript/entity/ThrownPotionData.java [hunk 1]
```diff
@@ -46,34 +47,33 @@
...
-							else
-								r.add(d);
-						}
-					}
-					return r;
+			return (types = Converters.convert((ItemType[]) exprs[0].getAll(), ItemType.class, t -> {
+				Material type = t.getMaterial();
+				// If the itemtype is a potion, lets make it a splash potion (required by Bukkit)
+				// Due to an issue with 1.12.2 and below, we have to force a lingering potion to be a splash potion
+				if (type == Material.POTION || (type == Material.LINGERING_POTION && RUNNING_LEGACY)) {
```
Comment (SnowPyon) [inline src/main/java/ch/njol/skript/entity/ThrownPotionData.java:63]: Don't do Material checks, put the corresponding alias in a constant through Aliases#javaItemtype instead.

File: src/main/java/ch/njol/skript/entity/ThrownPotionData.java [hunk 2]
```diff
@@ -46,34 +47,33 @@
...
+					ItemMeta meta = t.getItemMeta();
+					ItemType itemType = new ItemType(Material.SPLASH_POTION);
```
Comment (SnowPyon) [inline src/main/java/ch/njol/skript/entity/ThrownPotionData.java:65]: Same goes here, get the ItemType through Aliases#javaItemtype

File: src/main/java/ch/njol/skript/entity/ThrownPotionData.java (hunk 1 above)
Comment (ShaneBeee) [reply src/main/java/ch/njol/skript/entity/ThrownPotionData.java:63]: Ah yes you're right, totally forgot about that thanks. Ill switch them up.

Comment (ShaneBeee) [reply src/main/java/ch/njol/skript/entity/ThrownPotionData.java]: Ok these have now been changed!

**Pull request merged**